Fishing toolboxes (and maint) now come with a paper slip containing fishing-related tips. (#85279)
tgstation/tgstation: strings/fishing_tips.txt

Did you know? The Freshness jar of natural baits actually contains kronkaine.

> Legends say The Dude once lost hiswallet inside a toilet.

Reviewer: ```suggestion
Legends say The Dude once lost his wallet inside a toilet.
```

You could buy fish dirt cheap from the black market, but you probably won't get anything actually fresh.
Rescue hooks can be used to snag in other people. Good for getting husks out of plasma rivers.
Pufferfish are known for the slow yet lethal poison they store inside their liver.
Magnet hooks are great to fish anything that's NOT a fish.
You can perform scanning experiments with an experi-scanner and some fish. They'll give sci some extra techweb points and get you more modules for the fishing portal generator. 
Fish scanning experiments can be automatically performed with an advanced fishing rod. However to print one you need to have at least completed the first one.
Advanced fishing rods come pre-equipped with a unexhaustible bait that ignores bait preferences of the fish, letting you catch'em all.

> Don't expect to be able to fish much without a bait.

Reviewer: ```suggestion
Don't expect to be able to fish much without using bait.
```

You can fish at the holodeck beach, if you don't mind the fish being quite fake.
You may find worms by digging through sand and snow.
You can revive fish by using a Lazarus Injector on them. However, using Strange Reagent would be a smarter option here.
You can feed fish outside of an aquarium by tapping them with a can of fish feed.
More fishing rods and fish cases can be printed at the autolathe.
Seeking alternative ways to catch fish without bothering to do it yourself? Explosives can be thrown at fishing spots to get several (dead) ones.
You can raise lobstrosities from chasm chrabs. However, only juvenile lobstrosities can be tamed with spare limbs or lavaloop fish.

> Lavaloop fish make for dangerous yet somewhat throwing weapons against big fauna.

Reviewer: ```suggestion
Lavaloop fish make for dangerous yet somewhat effective throwing weapons against big fauna.
```
I assume this is what you meant.

The fishing portal generator has different modules, all of which can be unlocked by completing fish scanning experiments... except for the Syndicate one...

> Several fish have different traits, some of which can influence on how to catch them. Carnivore fish will ignore dough balls and herbivore ones anything that's not from botany.

Reviewer: ```suggestion
Fish traits influence how you can catch them. Carnivore fish will ignore dough balls, and herbivore fish ignore anything that's not from botany.
```

Telescopic fishing rods can be bought from cargo.
Once grown from chrabs and tamed, lobstrosities can be heeded to fish on fishing spots for you.

> Aquariums can be upgraded to bioelectricity generators can a specific kit. From there, you can add electric-generating like the anxious zip zap to generate power.

Reviewer: ```suggestion
Aquariums can be upgraded to bioelectricity generators can a specific kit. From there, you can add electric-generating fish like the anxious zip zap to generate power.
```

Getting better at fishing will net you some small additional advantages, such as receiving more information when examining a fish or a fishing spot.

> Size and weight of a fish can influence of many reagents and fillets you can get out of them, their force as a wapon and how slow they can be to carry around.

Reviewer: ```suggestion
The size and weight of a fish can influence the amount of reagents and fillets you can harvest from them, their force as a weapon and how easy it is to store them in containers.
```

Author: > I couldn't find anything in the code to suggest they slow you down?

Because it's from an upcoming PR.

Reviewer: ooooh

While most fish make for shoddy weapons, a few, like the swordfish and the chainsawfish, can be quite powerful if grown right.

> Fish can be sold to cargo. While most don't sell for over a couple hundred credits, the biggest ones can quite well sell for over 1k each.

Reviewer: ```suggestion
Fish can be sold to cargo. While most don't sell for very much, the biggest ones can be sold for a considerable bounty.
```

Being non-specific helps future proof these tips

The meat of fish like swordfish, salmon and pufferfish (weren't they poisonous) make for quality ingredients that boost the quality of the end product.
Stingrays are known for their stinger, which they use to deliver venom to hostiles.
Some species of fish can be bred into new species under the right conditions.
Most fish don't survive outside water, so get them somewhere safe like an aquarium or a fish case, or even a moisture trap?!

> No matter how you look at it, most people won't care about fishing. Think of it as a little funny niche.

Reviewer: ```suggestion
No matter how you look at it, most people won't care about fishing. Don't let that stop you. They're just jealous.
```

We should cement these in the game world. Talking about 'niches' kind of feels a little too OOC.

> To fish on ice you have to puncture the ice layer with a pick or shover first.

Reviewer: ```suggestion
To fish on ice you have to puncture the ice layer with a pick or shovel first.
```

Depending on the kind of fish inside it, an aquarium can improve the beauty of the room or worsen it.

> Almost all fish can be ground in an All-in-one-Grinder. Yeah, that includes those that wouldn't theorically  fit in a blender.

Reviewer: ```suggestion
Almost all fish can be ground in an All-in-one-Grinder. Don't think too hard about how you're fitting a giant tuna fish into a blender. Nanotrasen technology is weird like that.
```
